### ISSUE TITLE
[Bug]: [bulk insert v2]bulk insert with binlog keep pending and progress is a negative number

### ISSUE THREAD

**Issue by zhuwenxing** [milvus-io/milvus#31221]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:master
- Deployment mode(standalone or cluster):
- MQ type(rocksmq, pulsar or kafka):    
- SDK version(e.g. pymilvus v2.0.0rc2):
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

```
[2024-03-13 11:09:59 - DEBUG - ci_test]: 
method: post, 
url: http://10.104.19.197:19530/v2/vectordb/jobs/import/create, 
cost time: 0.06859207153320312, 
header: {'Content-Type': 'application/json', 'Authorization': 'Bearer root:Milvus', 'RequestId': '2d3a85b4-e0e7-11ee-aefc-acde48001122'}, 
payload: {"collectionName": "test_collection_2024_03_13_11_09_53_068467GWXPiSDD_restore", "files": [["milvus-bucket/file/insert_log/448343692569963162/", "milvus-bucket/file/delta_log/448343692569963162/"]], "options": {"backup": "true"}}, 
response: {"code":200,"data":{"jobID":"448343692569963550"}} (milvus.py:17)
[2024-03-13 11:09:59 - DEBUG - urllib3.connectionpool]: Starting new HTTP connection (1): 10.104.19.197:19530 (connectionpool.py:245)
[2024-03-13 11:09:59 - DEBUG - urllib3.connectionpool]: http://10.104.19.197:19530 "POST /v2/vectordb/jobs/import/list HTTP/1.1" 200 102 (connectionpool.py:547)
[2024-03-13 11:09:59 - DEBUG - ci_test]: 
method: post, 
url: http://10.104.19.197:19530/v2/vectordb/jobs/import/list, 
cost time: 0.07337498664855957, 
header: {'Content-Type': 'application/json', 'Authorization': 'Bearer root:Milvus', 'RequestId': '2d451150-e0e7-11ee-aefc-acde48001122'}, 
payload: {"collectionName": "test_collection_2024_03_13_11_09_53_068467GWXPiSDD_restore", "dbName": "default"}, 
response: {"code":200,"data":[{"jobID":"448343692569963550","progress":-9223372036854775808,"state":"Pending"}]} (milvus.py:17)
[2024-03-13 11:09:59 - DEBUG - urllib3.connectionpool]: Starting new HTTP connection (1): 10.104.19.197:19530 (connectionpool.py:245)
[2024-03-13 11:09:59 - DEBUG - urllib3.connectionpool]: http://10.104.19.197:19530 "POST /v2/vectordb/jobs/import/get_progress HTTP/1.1" 200 100 (connectionpool.py:547)
[2024-03-13 11:09:59 - DEBUG - ci_test]: 
method: post, 
url: http://10.104.19.197:19530/v2/vectordb/jobs/import/get_progress, 
cost time: 0.03721117973327637, 
header: {'Content-Type': 'application/json', 'Authorization': 'Bearer root:Milvus', 'RequestId': '2d5054de-e0e7-11ee-aefc-acde48001122'}, 
payload: {"jobID": "448343692569963550"}, 
response: {"code":200,"data":{"jobID":"448343692569963550","progress":-9223372036854775808,"state":"Pending"}} (milvus.py:17)
```

### Expected Behavior

success or report an error

### Milvus Log

[bulk-insert.log](https://github.com/milvus-io/milvus/files/14581754/bulk-insert.log)

**Comment by zhuwenxing** [comment]
/assign @bigsheeper 
PTAL

**Comment by bigsheeper** [comment]
The `test_job_import_binlog_file_type case` does not generate binlog because there is no flush before the import. I will modify it so that an error is reported when there is no binlog. @zhuwenxing

**Comment by bigsheeper** [comment]
fixed
/assign @zhuwenxing

**Comment by zhuwenxing** [comment]
![image](https://github.com/milvus-io/milvus/assets/12268675/4945aa08-6df2-4514-8f26-d43c2d8ad20b)

when binlog exists, api still reports error.
```
[2024-03-15 16:15:12 - DEBUG - ci_test]: 
method: post, 
url: http://10.104.26.83:19530/v2/vectordb/jobs/import/create, 
cost time: 0.08263182640075684, 
header: {'Content-Type': 'application/json', 'Authorization': 'Bearer root:Milvus', 'RequestId': '252ac104-e2a4-11ee-8b9e-acde48001122'}, 
payload: {"collectionName": "test_collection_2024_03_15_16_14_37_328286uvdCbXfQ_restore", "files": [["milvus-bucket/file/insert_log/448394271389851755/"]], "options": {"backup": "true"}, "dbName": "default"}, 
response: {"code":2100,"message":"list binlogs failed, err=insert binlog prefix does not exist, path=milvus-bucket/file/insert_log/448394271389851755/: importing data failed"}
```

**Comment by bigsheeper** [comment]
please help to make a verify @zhuwenxing

**Comment by zhuwenxing** [comment]
verified and fixed in `master-20240317-17b49389`
